tile_generator/templates/jobs/deploy-all.sh.erb

```diff
@@ -174,13 +179,12 @@ function add_env_vars() {
 	{% endfor %}
 	{% endfor %}
 
+	# Addresses and Properties Provided by bosh links
 	{% for release in context.releases.values() if release.consumes %}
 	{% for link in release.consumes %}
-	<% if_link('{{ link }}') do |link| %>
```

Comment:
Why is this not need anymore?

Reply:
While processing the links we always want a value set, if link content is present to set it, and if missing to unset it.